src/plugins/data/common/search/types.ts

```diff
@@ -37,7 +38,7 @@ export interface ISearchClient {
   extend: ISearchExtendGeneric;
 }
 
-export interface IKibanaSearchResponse<RawResponse = any> {
+export interface IKibanaSearchResponse<RawResponse = any> extends BatchResultBase {
```

Comment:
im a bit confused by this. why is the search response extending from batchresultbase ?

Reply:
Because the response now contains the `meta` property

Reply:
even when using it without batching ?